### PR TITLE
Image gallery with improved features using js.

### PR DESCRIPTION
**Pull request by Aarti002**
issue #386 solved 
Sorry for the delay actually  there were some problems in the project , i hope this would be okay now please review and let me know if something is wrong or missing @991rajat

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (991rajat) [inline .vscode/settings.json:2]: Remove this file.

**Comment by 991rajat** [comment]
Your project is not visible in project section. Add the project to script.js so that it is visible.

**Comment by Aarti002** [comment]
sorry @991rajat but i didn't get you i mean how am i suppose to add whole project in script.js?
html file--> public/imageGallery.html
css file -->assets/css/imageGallery.css
js file -->assets/js/imageGallery.js
all images -->assets/Images/imageGallery/images
i guess they are at correct place, right?

**Comment by 991rajat** [comment]
See the script.js in there.
Add a new project like this
{
projectName: {NAME},
projectImage: {GIF path to assets},
projectUrl: {Path to project},
}

**Comment by Aarti002** [comment]
@991rajat i added but there is some conflict, how to resolve this?

**Pull request merged**